packages/bruno-app/src/components/Sidebar/NewRequest/index.js

```diff
@@ -58,6 +59,12 @@ const NewRequest = ({ collection, item, isEphemeral, onClose }) => {
           name: 'requestName',
           message: `The request names - collection and folder is reserved in bruno`,
           test: (value) => {
+            if (fileNameRegex.test(value) === false) {
```

Comment:
Can you please make this check platform-specific?
```suggestion
            if (isWindowsOS() && fileNameRegex.test(value) === false) {
```

Reply:
I don't think that's correct way to go
https://github.com/usebruno/bruno/pull/2788#discussion_r1710103940

Reply:
Hello [sanjai0py](https://github.com/sanjai0py), 
As [pietrygamat](https://github.com/pietrygamat) mentioned, I updated the validation to be uniform across all platforms in this commit [68df1fca9aaff6344274702284db1ca7301d2acc](https://github.com/usebruno/bruno/pull/2788/commits/68df1fca9aaff6344274702284db1ca7301d2acc).

Would you prefer that I roll back this commit to make the validation platform-specific again?

Thank you for your guidance!

Reply:
The best way to handle this would most likely be, to replace forbidden characters in the filename when saving the request and putting the correct name into the Bru file contents

Reply:
Hey thanks Pietrygamat and Its-treason for correcting my mistakes. I agree with your comments.